### PR TITLE
1776/agency report barrister

### PR DESCRIPTION
**Pull request by tomlovesgithub**
## What's included?
this includes all candidates who have declared barrister experience in the BSB report, both on the admin site, and in the download.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
[Go to this exercise](https://jac-admin-develop--pr1915-1776-agency-report-b-pymsu4ql.web.app/exercise/11bv7ZTyyKhoqGYAPN7Y/reports/agency#bsb) under reports > agency report > BSB
- Under BSB ensure all candidates who have selected yes to barrister appear.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by HalcyonJAC — COMMENTED** [review]
Now on apply we collect the date a candidate called to the Bar and store it in the field `calledToBarDate`. But the `bsbDate` will be populated from the field `date` on digital-platform. Not sure which field we should use to show in the BSB table.
@tomlovesgithub,  could you double-check it, please?

File: src/views/Exercise/Reports/Agency.vue [hunk 1]
```diff
@@ -221,9 +221,18 @@
...
+                    v-if="candidate.bsbDate"
+                    class="govuk-table__cell"
+                  >
                     {{ candidate.bsbDate | formatDate('long') }}
                   </td>
+                  <td
+                    v-else
+                    class="govuk-table__cell"
+                  >
+                    {{ 'None given' }}
```
Comment (warrensearle) [inline src/views/Exercise/Reports/Agency.vue:234]: 😄 Do you need the double braces?

**Review by warrensearle — CHANGES_REQUESTED** [review]
One small tweak needed 😄

**Review by andrew-isaac — APPROVED** [review]

**Review by warrensearle — APPROVED** [review]
👍

**Pull request merged**